components/script/task_queue.rs

```diff
@@ -81,20 +87,16 @@ impl<T: QueuedTaskConversion> TaskQueue<T> {
 
         for msg in to_be_throttled {
             // Categorize tasks per task queue.
-            let (worker, category, boxed, pipeline_id) = match msg.into_queued_task() {
-                Some((worker, category, boxed, pipeline_id)) => (worker, category, boxed, pipeline_id),
+            let (worker, category, boxed, pipeline_id, task_source) = match msg.into_queued_task() {
+                Some(queued_task) =>
+                    queued_task,
```

Comment:
nit: I *think* this one would belong on the same line right after the `=>`, or has `rustfmt` formatted it that way?